v22.1/performance-benchmarking-with-tpcc-local-multiregion.md

```diff
@@ -37,6 +37,12 @@ cockroach demo --global --nodes 9 --no-example-database --insecure
 
 This simulated multi-region deployment will take advantage of CockroachDB's [multi-region SQL statements](multiregion-overview.html) to deliver improved ease of use and performance.
 
+{{site.data.alerts.callout_info}}
+You must use the IP address shown at the SQL prompt to run the following steps.
```

Comment:
The added callout looks good as-is.

One suggestion to clarify further (take it or leave it): add what the `cockroach demo` response looks like and include the sample IP address

Reply:
Thanks Lauren!  I think I'll pass for now as I would rather users refer to the `cockroach demo` docs which have that info